content/schedule/activities.md

```diff
@@ -9,4 +9,64 @@ people: all
 ---
 
 ## NWERC activities
-TBD
+The following NWERC excursions are available for Saturday, November 23. Each activity has limited capacity, so **full = full**. You must **wear your badge** to gain entrance to each venue.
+
+All activities take place between 15:00 and 18:00, though museums listed below **close at 17:00**.
+
+
+Please note that these excursions are **self-guided**, allowing you to visit your chosen activity at your own pace. All activities are within a 30-minute walking distance of the contest site. Dinner is served in the Flux building from 18:00 until 20:00, so be sure to come by on time after your activity.
+
+The registration form is available through a link in the information email sent on 02/11/2024.
```

Comment:
Do we have a deadline for filling in this form? If we do, then we should mention that here and on the form :slightly_smiling_face:

Reply:
added it to the form